config/environments/development.rb

```diff
@@ -70,7 +70,7 @@
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
-  # config.action_view.annotate_rendered_view_with_filenames = true
+  config.action_view.annotate_rendered_view_with_filenames = true
```

Comment:
@cdoyle-temple -- Can you explain what this is needed for?

Reply:
@ebtoner This should make debugging templates and view partials easier.  We can see what's being rendered directly in the html. @cdoyle-temple ?

Reply:
Ok, makes sense. Thanks for explaining!

Reply:
This helps when developing locally to see in the rendered code source (browser -> inspect) which template file generated each section of the page. Extremely handy shortcut that saves a lot of time with, especially with unfamiliar code-bases. We have it on manifold, the press, and library search.